tests/test_ast_symbol_extractor.py

```diff
@@ -586,12 +586,18 @@ def test_symbols_in_volume_names():
     import ast
     
     z = [ast.Try]
+    z.sort()
     """
     z = process_code_str(code)
     assert z.undeclared_symbols == set()
     assert z.post_process_symbols() == {
         "mm": {
-            "data": {"symbols_in_volume": {"ast.Try": {"line number": [4]}}},
+            "data": {
+                "symbols_in_volume": {
+                    "ast.Try": {"line number": [4]},
+                    "mm.z": {"line number": [5]},
```

Comment:
The line number for 'mm.z' in module volume and line number for 'mm.z' in the surface area do not match.

Why is 'mm.z' part of the volume for the module? It seems it should only be part of the surface area.

Reply:
To be fair, `mm.z` is used by the module. The mismatch is the difference between the line where `z` is defined (line 4) and where `z` is used (line 5).

This is what the results of the parsing is on master, which pick up the `mm.z` in the volume
```python
{'mm': {'data': {'symbols_in_volume': {'mm.ast': {'line number': [4]},
                                       'mm.z': {'line number': [5]}}},
        'type': 'module'},
 'mm.ast': {'data': {'shadows': 'ast'}, 'type': 'import'},
 'mm.z': {'data': {'lineno': 4}, 'type: 'constant'}}
```

Reply:
Got it, we don't necessarily know that 'z' was defined just before, it can just as easily be a symbol defined in module (say a function) and not 'used' in the module, in this case it is being used so we capture it as part of the volume.